src/components/v-mapbox-layer.js

```diff
@@ -63,10 +75,29 @@ export default {
           if(layerSource && !map.getStyle().layers.some(({ source }) => source === layerSource)) {
             map.removeSource(layerSource);
           }
+          if(this.clickable) {
+            map.off('click', layerId, this.clickFn);
+            map.off('mouseenter', layerId, this.mouseEnterFn);
+            map.off('mouseleave', layerId, this.mouseLeaveFn);
+          }
         }
       }
     },
 
+    clickFn(e) {
+      this.$emit('click', e);
+    },
+
+    mouseEnterFn() {
+      const map = this.getMap();
+      map.getCanvas().style.cursor = 'pointer';
```

Comment:
just an idea; maybe cursor style should be a prop the user can specify?

Reply:
may be we can handle this as a separate feature request ☺️